src/Mocap/MocapBodyDriveDebug.cpp

```
@@ -0,0 +1,295 @@
+void fkPoseIkJoints(
+    const std::array<std::array<float, 4>, PoseIK::kCanonicalRoles>& quats,
+    uint32_t resolvedMask,
+    const std::array<std::array<float, 3>, PoseIK::kLandmarkCount>& canonLmPts,
+    std::array<Vec3, PoseIK::kCanonicalRoles>& out)
+{
+    out.fill({0.f, 0.f, 0.f});
+    const Vec3 hip = {
+        (canonLmPts[23][0] + canonLmPts[24][0]) * 0.5f,
+        (canonLmPts[23][1] + canonLmPts[24][1]) * 0.5f,
+        (canonLmPts[23][2] + canonLmPts[24][2]) * 0.5f};
+    out[static_cast<size_t>(PoseIK::Hip)] = hip;
```

Cppcheck (2.21.0)

[error] Address of local auto-variable assigned to a function parameter.

(autoVariables)